app/Http/Controllers/ReviewController.php

```diff
@@ -16,6 +18,13 @@ public function __construct(ReviewService $service)
         $this->service = $service;
     }
 
+    public function update(DoctorRespondReviewRequest $request, $id)
+    {
+        $review = $this->service->show($id);
+        $this->service->update($request->all('respond'), $review->id);
```

Comment:
- Validate that the appointment belongs to this doctor
- Validate that he didn't respond before